docs/tables/aws_account_contact.md

```diff
@@ -0,0 +1,20 @@
+# Table: aws_account_contact
+
+We can store contact information about the primary account contact for your AWS account.
+
```

Comment:
Please add more info on what's required to use the `contact_account_id` column, e.g.,
```
To use this parameter, the caller must be an identity in the [organization's management account](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_getting-started_concepts.html#account) or a delegated administrator account. The specified account ID must also be a member account in the same organization. The organization must have [all features enabled](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_org_support-all-features.html), and the organization must have [trusted access](https://docs.aws.amazon.com/organizations/latest/userguide/using-orgs-trusted-access.html) enabled for the Account Management service, and optionally a [delegated admin](https://docs.aws.amazon.com/organizations/latest/userguide/using-orgs-delegated-admin.html) account assigned.
```

Taken from https://docs.aws.amazon.com/accounts/latest/reference/API_GetContactInformation.html#API_GetContactInformation_RequestBody